confluent_kafka/src/Consumer.c

```diff
@@ -386,8 +386,8 @@ static PyObject *Consumer_commit (Handle *self, PyObject *args,
                 return NULL;
         }
 
-	if (!PyArg_ParseTupleAndKeywords(args, kwargs, "|OOO", kws,
-					 &msg, &offsets, &async_o))
+	if (!PyArg_ParseTupleAndKeywords(args, kwargs, "|OOOO", kws,
+					 &msg, &offsets, &async_o, &async_o))
```

Comment:
That's quite nice, does it just overwrite based on which one is set?

Reply:
Yep, this works since ParseTupleAndKeywords returns borrowed references and does not increase the refcount for each matched object (which wouldn't work here).